tardis/tests/integration_tests/integration.yml

```diff
@@ -0,0 +1,35 @@
+atom_data_path: "~/Desktop/tardis/tardis-refdata/"
```

Comment:
@epassaro please change this to point to your refdata. As well as some other lines in this file.